84LargestRectangleinHistogram.py

```
@@ -0,0 +1,95 @@
+                else:
+                    count+=1
+            for j in range(i+1,n):
+                if heights[j]<heights[i]:
+                    break 
+                else:
+                    count+=1
+                    
+            max_area=max(heights[i]*count,max_area)
+        return max_area 
+# Precomputing pse, nse Time Complexity : O(3N) Space Complexity : O(4N)
+class Solution:
```

Ruff (0.12.2)

Redefinition of unused `Solution` from line 26

(F811)